app/assets/javascripts/controllers/container_dashboard/util/charts-mixin.js

```diff
@@ -48,17 +48,23 @@ angular.module('miq.util').factory('chartsMixin', function() {
 
   var processHeatmapData = function(heatmapsStruct, data) {
     if (data) {
-      heatmapsStruct.data = _.sortBy(data, 'percent').map(function(d) {
-        var percent = d.percent * 100;
-        var tooltip = "Node: " + d.node + "<br> Provider: " + d.provider + "<br> Usage: " + percent + "% in use of " +
-          d.total + " total";
+      var heatmapsStructData = data.map(function(d) {
+        var percent = 0;
+        var tooltip = "Node: " + d.node + "<br> Provider: " + d.provider
+        if (d.percent === null || d.total === null) {
+          tooltip += "<br> Usage: Unknown"
+        } else {
+          percent = d.percent
+          tooltip += "<br> Usage: " + (percent * 100).toFixed(0) + "% in use of " + d.total + " total";
+        }
 
         return {
           "id": d.id,
           "tooltip": tooltip,
-          "value": d.percent
+          "value": percent
         };
-      }).reverse()
+      })
+      heatmapsStruct.data = _.sortBy(heatmapsStructData, 'value').reverse()
```

Comment:
> Previously, if data.percent was null, it would get sorted last. Now, it will be sorted as if the value was 0. Is it intended?

yeah, nulls get sorted as first (because of reverse?) so i set them to 0...